Buildings/Resources/Scripts/Dymola/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/Amigo/Validation/PID.mos

```diff
@@ -0,0 +1,5 @@
+simulateModel("Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.Validation.PID", method="Cvode", tolerance=1e-06, resultFile="PID");
```

Comment:
delete the folder `Buildings/Resources/Scripts/Dymola/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/Amigo`.

Reply:
done